yolort/models/transform.py

```diff
@@ -283,17 +293,40 @@ def _resize_image_and_masks(
     return image, target
 
 
-def resize_boxes(boxes: Tensor, original_size: List[int], new_size: List[int]) -> Tensor:
-    ratios = [
-        torch.tensor(s, dtype=torch.float32, device=boxes.device) /
-        torch.tensor(s_orig, dtype=torch.float32, device=boxes.device)
-        for s, s_orig in zip(new_size, original_size)
-    ]
+def resize_boxes(boxes: Tensor, new_size: List[int], original_size: List[int], use_square_box=False) -> Tensor:
```

Comment:
This will be BC-breaking, we should be cautious about the modification of this interface here.